src_kt/java/arcs/storage/driver/Volatile.kt

```diff
@@ -176,7 +180,7 @@ internal data class VolatileEntry<Data : Any>(
  */
 internal class VolatileMemory {
   private val lock = Any()
-  private val entries = mutableMapOf<StorageKey, VolatileEntry<*>>()
+  private val entries: HashMap<StorageKey, VolatileEntry<*>> = hashMapOf()
```

Comment:
Does mutableMapOf() in Kotlin not return a HashMap?

Reply:
ah, it does... this was leftover from me wondering why my IDE wasn't cooperating. Will change this back.